x/twap/client/query_proto_wrap.go

```diff
@@ -18,7 +18,10 @@ type Querier struct {
 func (q Querier) ArithmeticTwap(ctx sdk.Context,
 	req queryproto.ArithmeticTwapRequest, // nolint: staticcheck
 ) (*queryproto.ArithmeticTwapResponse, error) {
-	if (req.EndTime == nil || *req.EndTime == time.Time{}) {
+	if req.EndTime == nil {
+		req.EndTime = &time.Time{}
+	}
+	if (*req.EndTime == time.Time{}) {
```

Comment:
Why did this get changed? Old behavior looks correct to me

Reply:
if `req.EndTime` is nil, the old behavior would try to dereference a nil pointer.